egov-bndry-mgmnt/src/server/utils/excelUtils.ts

```
@@ -0,0 +1,403 @@
+  const columnIndexesToBeFreezed: any = [];
+  const columnIndexesToBeHidden: any = [];
+  if (frozeCells) {
+    performUnfreezeCells(sheet, localizationMap, fileUrl);
+  }
+  if (frozeWholeSheet) {
+    performFreezeWholeSheet(sheet);
+  }
+  let columnsToBeFreezed: any[] = [];
+  let columnsToHide: any[] = [];
+  if (fileUrl) {
+    columnsToHide = ["CRS_BOUNDARY_CODE_OLD", ...schema?.columnsToHide];
```

Biome (2.1.2)

[error] Unsafe usage of optional chaining.

If it short-circuits with 'undefined' the evaluation will throw TypeError here:


(lint/correctness/noUnsafeOptionalChaining)